tests/phpunit/integration/tests/Migrations/Add_Media_Source_Editor.php

```diff
@@ -46,8 +47,11 @@ public function set_up(): void {
 	 */
 	public function test_migrate(): void {
 		$this->instance->migrate();
-		$term = $this->call_private_method( $this->instance, 'get_term' );
+		$term = $this->call_private_method( [ $this->instance, 'get_term' ] );
 
+		/**
+		 * @var WP_Term[] $terms
```

Comment:
Would this not be covered by - https://github.com/szepeviktor/phpstan-wordpress/blob/3934e622e8434092aef69761d684281812fef868/src/GetTermsDynamicFunctionReturnTypeExtension.php#L26

Reply:
Only if we add `$this->assertNotWPError( $terms );`

Reply:
Added.